packages/@react-spectrum/list/src/ListView.tsx

```diff
@@ -44,19 +44,28 @@ export function useListLayout<T>(state: ListState<T>) {
   return layout;
 }
 
-interface ListViewProps<T> extends CollectionBase<T>, DOMProps, AriaLabelingProps, StyleProps {
+interface ListViewProps<T> extends CollectionBase<T>, DOMProps, AriaLabelingProps, StyleProps, MultipleSelection {
+  /**
+   * Sets the amount of vertical padding within each cell.
+   * @default 'regular'
+   */
+  density?: 'compact' | 'regular' | 'spacious',
   isLoading?: boolean,
   renderEmptyState?: () => JSX.Element,
-  transitionDuration?: number
+  transitionDuration?: number,
+  onAction?: (key: Key) => void
 }
 
 function ListView<T extends object>(props: ListViewProps<T>, ref: DOMRef<HTMLDivElement>) {
   let {
-    transitionDuration = 0
+    density = 'regular',
+    transitionDuration = 0,
+    onAction
   } = props;
   let domRef = useDOMRef(ref);
   let {collection} = useListState(props);
   let formatMessage = useMessageFormatter(intlMessages);
+  let [selectionMode, setSelectionMode] = useState(false);
```

Comment:
What is this `selectionMode` used for? Is it different from the `state.selectionManager`'s selectionMode?